src/Resources/download/index.html

```diff
@@ -215,16 +215,6 @@ <h2 class="text-center">Open Source IoT Development Framework for ESP32</h2>
                 </form>
             </div>
 
-	        <div class="download-button">
-                <form method="get" action="https://dl.espressif.com/dl/idf-installer/esp-idf-tools-setup-offline-5.4.exe">
-                    <button class="button-offline">
-                        <i class="fa fa-download" aria-hidden="true"></i>
-                        <div>ESP-IDF v5.4 - Offline Installer</div>
-                        <div>Windows 10, 11</div>
-                        <div>Size: 1.72 GB</div>
-                    </button>
-                </form>
-            </div>
```

Comment:
I believe this removal is caused by automatic script. Thats probably because it was added manually by me here: https://github.com/espressif/idf-installer/pull/303/files

It seems to me that the missing piece to the automatic div addition is to append `5.4` to this list: https://github.com/espressif/idf-installer/blob/main/.github/workflows/build-installer.yml#L33

Reply:
@dobairoland